src/pcre2grep.c

```diff
@@ -99,10 +99,9 @@ POSSIBILITY OF SUCH DAMAGE.
 #include <bzlib.h>
 #endif
 
-#include "pcre2_util.h"
-
 #define PCRE2_CODE_UNIT_WIDTH 8
 #include "pcre2.h"
```

Comment:
The original idea of pcre2grep was that it was something an external PCRE2 user could have written. In that spirit, it is not right to #include any of the internal pcre2....h  files such as pcre2_util.h or pcre2_internal.h.

Reply:
Ah I see. So it's OK for pcre2test to use those, but pcre2grep is an "external" client of the API.

I can do that. It would be useful to have assertions though...